source/modules/MEI.visual.xml

```diff
@@ -1254,6 +1253,7 @@
       <memberOf key="att.staffLoc.pitched"/>
       <memberOf key="att.typography"/>
       <memberOf key="att.width"/>
+      <memberOf key="att.visibility"/>
```

Comment:
```suggestion
      <memberOf key="att.visibility"/>
      <memberOf key="att.width"/>
```

Reply:
Alphabetical order

Reply:
Out of curiosity, what's the point of an invisible multiRest? It would not cover any more measure space than an mSpace, right?